test/new-e2e/tests/windows/install-test/assert.go

```diff
@@ -42,3 +42,18 @@ func AssertInstalledUserInRegistry(t *testing.T, host *components.RemoteHost, ex
 
 	return true
 }
+
+// AssertUserRights checks the user has the expected user rights
+func AssertUserRights(t *testing.T, host *components.RemoteHost, username string) bool {
+	expectedRights := []string{
+		"SeServiceLogonRight",
+		"SeDenyInteractiveLogonRight",
+		"SeDenyNetworkLogonRight",
+		"SeDenyRemoteInteractiveLogonRight",
+	}
+	actualRights, err := windows.GetUserRightsForUser(host, username)
+	if !assert.NoError(t, err, "should get user rights") {
+		return false
+	}
+	return assert.ElementsMatch(t, expectedRights, actualRights, "user %s should have user rights", username)
```

Comment:
Not sure ElementMatch logic. Will it succeed if there are more actual rights? Is that a problem in general and for future if it is?